kernel-commits.mk

```diff
@@ -2,8 +2,12 @@ KERNEL_COMMIT_amd64_v5.10.186_generic = d61682724485
 KERNEL_COMMIT_amd64_v6.1.38_generic = 8859f43ee8c9
 KERNEL_COMMIT_amd64_v6.1.38_rt = 37bc37278441
 KERNEL_COMMIT_amd64_v6.1.68_generic = 45e0ac394428
+KERNEL_COMMIT_amd64_v6.1.105_rt = 8fe1d58eff31
+KERNEL_COMMIT_amd64_v6.1.106_generic = ee5e09825b81
 KERNEL_COMMIT_arm64_v5.10.104_nvidia = 7b9a55e0b659
 KERNEL_COMMIT_arm64_v5.10.192_nvidia = c2b4176f82b1
 KERNEL_COMMIT_arm64_v5.10.186_generic = 804663a82829
 KERNEL_COMMIT_arm64_v6.1.38_generic = acbfadebeb76
+KERNEL_COMMIT_arm64_v6.1.106_generic = ee5e09825b81
```

Comment:
same commit SHAs seem very suspecious

Reply:
arm not yet pushed, this is reflected in the description of the PR.